code/modules/vending/cigarette.dm

```diff
@@ -13,7 +13,8 @@
 					/obj/item/lighter/greyscale = 4,
 					/obj/item/storage/fancy/rollingpapers = 5)
 	contraband = list(/obj/item/lighter = 3,
-					/obj/item/clothing/mask/vape = 5)
+					/obj/item/clothing/mask/vape = 5,
+					/obj/item/storage/fancy/cigarettes/cigpack_shadyjims = 3)
```

Comment:
This is a bad idea.

Reply:
How is it a bad idea, what kind of reagents does a pack of shady jims have in it? I've legitimately never seen them because it's unobtainable outside a contraband crate, and for a 750 bounty, thats was literally unobtainable otherwise, I figured adding it to a hacked vendor list would be the easiest way to make it relevant.

Reply:
I just looked it up, I'm not sure what ammonia does when injested, but plant-b-gone and toxin are just, well, toxins. Smoking kills, this just kills faster. It -is- in contraband after all. People already smoke other stuff for lipocide. You can get pure toxin and plant-b-gone easily enough, but the alternatives are to remove the bounty, or reduce the amount of packs you can get from the vendor.

Reply:
Eh, keep it to 1, or add a probability on initialize for the vendor to have those cigs.

Reply:
I can do that, yea.